src/Mount.cpp

```diff
@@ -1689,7 +1974,7 @@ String Mount::getStatusString() {
     status = "Idle,";
   }
 
-  String disp = "-----,";
+  String disp = "------,";
```

Comment:
Won't this break the `:GX` meade command?

Reply:
No. It's not an official LX200 command and only OATControl uses it, and it doesn't care if there are extras.